certif/translations/en.json

```
@@ -479,6 +475,61 @@
         }
       }
     },
+    "session-finalization": {
+      "actions": {
+        "finalise": "Finalise"
+      },
+      "title": "Finalise the session {sessionId}",
+      "return-to": "Return to the session",
+      "reporting": {
+        "completed-reports-information": {
+          "description": "Finished certification(s)",
+          "extra-information": "List of candidates who have finished their certification exam, ordered by birth name, with a link to add a reporting if needed",
+          "table": {
+            "labels": {
+              "screen": "End of exam screen seen"
+            }
+          }
+        },
+        "description": "For the reporting to be taken into account, you must use the appropriate reporting category (examples : C1, C2, etc).",
+        "uncompleted-reports-information": {
+          "description" : "These candidates haven’t finished their Pix certification exam or the invigilator has ended their exam",
+          "extra-information": "List of candidates who haven’t finished their certification exam, ordered by birth name, with a link to add a reporting if needed and a drop-down list enabling you to select the reason for abandonment",
+          "table": {
+            "labels": {
+              "abandonment": "Abandonment : lack of time or early departure",
+              "abandonment-reason": "Reason for abandonment",
+              "technical-problem": "Technical problem"
+            },
+            "tooltip": {
+              "abandonment": {
+                "enough-time-description": "The candidate didn’t have enough time to answer all the questions",
+                "left-deliberately-description": "The candidate left deliberately before the end of the exam"
+              },
+              "extra-information": "Reason for abandonment's description",
+              "technical-problem-description": "The candidate encountered a technical problem that prevented him from finishing his exam"
+            }
+          }
+        },
+        "table": {
+          "labels": {
+            "certification-number": "Certification No.",
+            "reporting": "Reporting"
+          },
+          "reporting-count": "{reportingsCount, plural,one {1 reporting} other {# reportings}}"
+        },
+        "title": "Reporting: Signal, for each candidate, the incidents inputted into the Incident report"
```

```suggestion
        "title": "Reporting: Signal, for each candidate, the incidents inputted into the Incident Report"
         "title": "Reporting: Signal, for each candidate, the incidents inputted into the incident report"
```

C'est pas un peu bizarre cette majuscule sur un seul des deux mots ? 🤔

---

j'imagine que ça doit être une coquille pendant la trad, je corrige ça !